packages/server/logging/apolloSubscriptions.ts

```diff
@@ -105,13 +105,7 @@ export function logSubscriptionOperation(params: {
       if (error instanceof BaseError) {
         errorLogger = errorLogger.child({ ...error.info() })
       }
-      if (shouldLogAsInfoLevel(error)) {
-        errorLogger.info({ err: error }, errMsg)
-      } else if (shouldLogAsWarnLevel(error)) {
-        errorLogger.warn({ err: error }, errMsg)
-      } else {
-        errorLogger.error({ err: error }, errMsg)
-      }
+      logWithErr(errorLogger, error)(errMsg)
```

Comment:
is this ever going to be invoked without all of the 3 params available at the same time? I don't see the point of this being a Higher Order Function